src/lib/components/textbadge/TextBadge.component.js

```diff
@@ -0,0 +1,30 @@
+//@flow
+import React from "react";
+import styled from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import { getThemeVariantSelector, getThemePropSelector } from '../../utils';
+import type { Variant } from "../constants";
+
+const StyledTextBadge = styled.span`
+  background-color: ${getThemeVariantSelector};
+  color: ${getThemePropSelector('textPrimary')};
+  padding: 2px 0.6rem;
+  border-radius: 4px;
+  font-size: 0.9rem;
+  font-weight: ${defaultTheme.fontWeight.bold};
+  margin: 0 ${defaultTheme.padding.smaller} 0 ${defaultTheme.padding.smaller};
+`;
+
+type Props = {
+  text: string,
+  variant?: Variant,
```

Comment:
IMO, We should not use the predefined Variant anymore. 

I suggest we define the following variant:
`variant?: 'statusHealthy' | 'statusWarning' | 'statusCritical' | 'infoPrimary'| 'infoSecondary' `

And default on should be `infoPrimary`

Reply:
I didn't understand, should I change the variant type like this:

`export type Variant = 'statusHealthy' | 'statusWarning' | 'statusCritical' | 'infoPrimary'| 'infoSecondary';`

or only the Props of TextBadge like this :
```type Props = {
  text: string,
  variant?: 'statusHealthy' | 'statusWarning' | 'statusCritical' | 'infoPrimary'| 'infoSecondary'
};
```

If I change the variant type I will have to change others components stories (because their variants will no longer be good)
Thanks!

Reply:
@ChengYanJin I didn't understand, should I change the variant type like this:

`export type Variant = 'statusHealthy' | 'statusWarning' | 'statusCritical' | 'infoPrimary'| 'infoSecondary';`

or only the Props of TextBadge like this :
```type Props = {
  text: string,
  variant?: 'statusHealthy' | 'statusWarning' | 'statusCritical' | 'infoPrimary'| 'infoSecondary'
};
```

If I change the variant type I will have to change others components stories (because their variants will no longer be good)
Thanks!

Reply:
only the Props of TextBadge:

  text: string,
  variant?: 'statusHealthy' | 'statusWarning' | 'statusCritical' | 'infoPrimary'| 'infoSecondary'
};

Reply:
done